### PR TITLE
Remove dependency on platzky

### PR DESCRIPTION
**Pull request by oskarrrrrrr**
Moved `platzky` code into the project. Moved unit tests but discarded e2e tests because they seem redundant. Discarded config template and readme, merged dependencies (`pyproject.toml`).

This is a first step towards removing the dependency on `platzky` and instead building a completely custom solution which gives us more control.

### CODE REVIEW

**Comment by raven-wing** [comment]
One concern - `pyright ignore`... We have to come up with sth here, because using this annotation makes this tool useless (we just could copy it everywhere)...

So the questions are:
- if ignored - should be commented 'why is it ignored'?
- make it strict ('not passing? not commiting?')
- is there any option for pyright that skips `ignore` annotations?

**Comment by oskarrrrrrr** [comment]
Generally, sometimes we will need `pyright: ignore`, because some things are just typed incorrectly or are too dynamic to be detected correctly by `pyright`. For example, if we use a library with a function with this kind of annotation: `def f(a: str = None)` then `pyright` will not be happy, because obviously `a` is of type `str | None`. Since it's in some library, there is little we can do. I've definitely encountered this kind of bug before. Another common problem is when something is too dynamic, maybe some attributes are added to a class magically instead of using `def` and in such cases it's very hard/impossible to be sure that the attribute that we use actually exists.
So, we can't disable adding `pyright: ignore` completetly. We definitely could add comments.

The way `pyright` is setup now will cause errors whenever `pyright ignore` is used unnecessarily (`reportUnnecessaryTypeIgnoreComment = true`).

For now I've disabled type checking for `platzky` anyway and only fixed the errors that I had too. I didn't want to change the original code, because it would be easy to break something and the diff would be useless as it's all new code. My plan was to enable `pyright` for `platzky` in the next change and address the errors.

**Comment by oskarrrrrrr** [comment]
Can we ignore coverage for now?

File: goodmap/platzky/blog/blog.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    @blog.route('/', methods=["GET"])
+    def index():
+        lang = locale()
+        return render_template("blog.html", posts=db.get_all_posts(lang))
+
+    @blog.route('/feed', methods=["GET"])
+    def get_feed():
+        lang = locale()
+        response = make_response(render_template("feed.xml", posts=db.get_all_posts(lang)))
+        response.headers["Content-Type"] = "application/xml"
```
Comment (kacper-cyra) [inline goodmap/platzky/blog/blog.py:31]: Do we need blog mechanics?

**Comment by raven-wing** [comment]
As discussed:
- platzky will be ignored in coverage
- ignores will be introduced to code temporarily
- cleanup of ignores has another issue #41

**Pull request merged**